sell-loot.lic

```diff
@@ -80,15 +88,15 @@ class SellLoot
       .each { |currency| fput "exchange all #{currency} for #{exchange_to}" }
   end
 
-  def deposit_coins(keep_copper)
+  def deposit_coins(keep_money_by_currency)
```

Comment:
❓ Could, or should, this be removed in favor of the function in [common-money](https://github.com/rpherbig/dr-scripts/blob/eb5fec5570e7281f32f3408672a1b9d288097e2a/common-money.lic#L119)?